gallery/tutorials/pipeline_demo.py

```diff
@@ -216,13 +232,14 @@
 
 from aspire.abinitio import CLSyncVoting
 from aspire.source import OrientedSource
+from aspire.utils import Rotation
 
 # Stash true rotations for later comparison
-true_rotations = src.rotations[:n_classes]
+true_rotations = Rotation(src.rotations[:n_classes])
 
 # For this low resolution example we will customize the ``CLSyncVoting``
```

Comment:
Appears we are no longer customizing

Reply:
Oh jeez. Thanks for catching that. Changed.